flink-connectors/flink-connector-rabbitmq/src/test/java/org/apache/flink/streaming/connectors/rabbitmq/RMQSourceITCase.java

```diff
@@ -112,7 +117,8 @@ public void setUp() throws Exception {
     }
 
     @Test
-    public void testStopWithSavepoint() throws Exception {
+    void testStopWithSavepoint(@InjectClusterClient RestClusterClient<?> clusterClient)
```

Comment:
```suggestion
    void testStopWithSavepoint(@InjectClusterClient RestClusterClient<?> clusterClient, @TempDir Path tmp)
```
nit: we could inject the `@TempDir` as a parameter since it's only used once in the test class.